### ISSUE TITLE
Can't update from 4.5.3 to 4.6.0 (dyld: lazy symbol binding failed: Symbol not found: _pthread_jit_write_protect_supported_np)

### ISSUE THREAD

**Issue by molul** [strapi/strapi#15614]
### Required System information

- Node.js version: v18.12.1
- NPM version: 9.1.3
- Strapi version: 4.6.0 (updating from 4.5.3)
- Database: sqlite
- Operating system: MacOS

### Describe the bug

I followed the update guide, but Strapi can't launch when I run "npm run develop".

### Steps to reproduce the behavior

I followed [the update guide](https://docs.strapi.io/developer-docs/latest/update-migration-guides/update-version.html#step-1-upgrade-the-dependencies). No migration needed as I was in 4.5.3. So the steps have been:

1- Change package.json. Mine's dependencies were:
``` 
"dependencies": {
    "@strapi/plugin-i18n": "4.5.3",
    "@strapi/plugin-users-permissions": "4.5.3",
    "@strapi/provider-email-nodemailer": "^4.5.3",
    "@strapi/strapi": "4.5.3",
    "better-sqlite3": "7.4.6",
    "node-postgres": "^0.6.2",
    "pg": "^8.8.0"
  },
```

And I changed it to:
```
  "dependencies": {
    "@strapi/plugin-i18n": "4.6.0",
    "@strapi/plugin-users-permissions": "4.6.0",
    "@strapi/provider-email-nodemailer": "^4.6.0",
    "@strapi/strapi": "4.6.0",
    "better-sqlite3": "7.4.6",
    "node-postgres": "^0.6.2",
    "pg": "^8.8.0"
  },
```

2-  Then I run "npm install", and "npm run build" after that, as said in the guide. So far so good.

3- Then I run "npm run develop" and got this message:
```
yarn run v1.22.19
$ strapi develop
dyld: lazy symbol binding failed: Symbol not found: _pthread_jit_write_protect_supported_np
  Referenced from: /Users/luis/Desktop/buscogrupo/repos/buscogrupo-cms/node_modules/@strapi/plugin-upload/node_modules/sharp/build/Release/../.././vendor/8.13.3/darwin-x64/lib/libvips-cpp.42.dylib
  Expected in: /usr/lib/libSystem.B.dylib

dyld: Symbol not found: _pthread_jit_write_protect_supported_np
  Referenced from: /Users/luis/Desktop/buscogrupo/repos/buscogrupo-cms/node_modules/@strapi/plugin-upload/node_modules/sharp/build/Release/../.././vendor/8.13.3/darwin-x64/lib/libvips-cpp.42.dylib
  Expected in: /usr/lib/libSystem.B.dylib
```

Also tried yarn develop. Obviously same result. Strapi can't launch, so I rolled back to 4.5.3.

### Expected behavior

I'd expect that "npm run build" would launch Strapi 4.6.0

**Comment by molul** [comment]
@Enache-Ionut  That didn't make the trick for me, unfortunately. I didn't have any sharp version installed. 

Nevertheless, I tried running these commands one after another (with the package.json dependencies modified as in the OP):

npm uninstall sharp (just in case)
npm install
npm run build
npm run develop

But I got the same message. I wonder if it's something about these three dependencies I have:
"better-sqlite3": "7.4.6",
"node-postgres": "^0.6.2",
"pg": "^8.8.0"

I'll try to make a bank Strapi project this weekend to see if the issue can be reproduced

**Comment by Boegie19** [comment]
Upsteam issue with sharp:
https://github.com/lovell/sharp/issues/3438

**Comment by derrickmehaffy** [comment]
Was able to confirm the issue is due to upstream dependency where the OS is MacOS Big Sur and lower

**Comment by PaulBratslavsky** [comment]
Thankt you @Boegie19 and @derrickmehaffy  Makes sense why I was not able to replicate the issue.  Running mac os ventura.

**Comment by molul** [comment]
@Boegie19 sorry for the noob question but I'm not sure if I can fix it myself already with the info in the link you provided (which I didn't fully understand) or if I should wait for some kind of instructions here ^_^U

Thanks in advance.

**Comment by derrickmehaffy** [comment]
> @Boegie19 sorry for the noob question but I'm not sure if I can fix it myself already with the info in the link you provided (which I didn't fully understand) or if I should wait for some kind of instructions here ^_^U
> 
> Thanks in advance.

He submitted a fix to the main repo, there are two ways you can resolve it for your project:

Use something like yarn resolutions to forcefully set the sharp version or upgrade your Mac beyond big sur.

**Comment by derrickmehaffy** [comment]
Alternatively I'm trying to get his fixed merged for the 4.6.1 release on Wednesday.

**Comment by Boegie19** [comment]
got merged so in 4.6.1 it should be fixed

**Comment by molul** [comment]
@Boegie19 awesome! Thank you very much :)
